app/AvailablePlugin/PrivacyIdeaAuthenticator/Model/PrivacyIdea.php

```diff
@@ -171,9 +171,11 @@ public function createToken($privacyIdeaAuthenticator, $coPersonId) {
     $params = array(
       'type' => 'totp',
       'genkey' => '1',
+      'user' => $identifier,
+      'realm' => $privacyIdeaAuthenticator['realm'],
       'optlen' => '6'
-    );
-    
+    );    
+
```

Comment:
Since this whitespace change isn't related to the fix, it shouldn't be included in the PR.